package.json

```diff
@@ -126,11 +132,12 @@
     "husky": "2.4.1",
     "inquirer": "3.3.0",
     "inquirer-confirm": "2.0.3",
-    "js-codemod": "cpojer/js-codemod#29dafed",
```

Comment:
why are we un-pinning from a specific sha?

Reply:
For some reason, it cannot find that SHA; despite me seeing it in GitHub. I have pinned to the very next commit's SHA to keep it as close to the original as possible.